tests/test_create_project.py

```diff
@@ -266,6 +266,31 @@ def test_generate_project_with_default_values(
         assert f"{param}={value}" in test_file_contents
 
 
+@parametrize_by_project_generation_params
+def test_generate_project_check_feature_store_output(
+    tmpdir, cloud, cicd_platform, include_feature_store
+):
+    """
+    Asserts the behavior of feature store-related artifacts when generating Stacks.
+    """
+    context = {
+        "project_name": TEST_PROJECT_NAME,
+        "cloud": cloud,
+        "cicd_platform": cicd_platform,
+        "include_feature_store": include_feature_store,
+    }
+    if cloud == "azure":
```

Comment:
Great catch. This was because the test I copied this from was testing that the default value is "azure" for cloud - here's the reasoning:
https://github.com/databricks/mlops-project-template/pull/74#discussion_r939063776

I added a comment to that effect in the right place and removed this check from here, as it's not relevant for this test.